linux_os/guide/services/ssh/ssh_client/ssh_client_use_approved_ciphers_ordered_stig/rule.yml

```diff
@@ -0,0 +1,29 @@
+{{%- if product == 'ubuntu2404' %}}
+{{%- set ssh_approved_ciphers = "aes256-gcm@openssh.com,aes128-gcm@openssh.com,aes256-ctr,aes128-ctr" %}}
+{{%- endif %}}
+documentation_complete: true
+
+title: 'Use Only FIPS 140-3 Validated Ciphers'
```

Comment:
As there are multiple rules with similar title I think it would be better to add "in SSH Configuration" or similar to the rule title.